.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,10 @@
+Please help potential reviewers to understand this pull request and speed up 
+the process by answering the following questions, where appropriate.
+
+# Why was this pull request created?
```

Comment:
What's this supposed to ask for? 'because there was a trello card'? Or 'xy crashes when using z'?

I hope it's the latter, but then I'd describe this as "Why is this change necessary?"